### PR TITLE
Add exoscale docs

### LINKED ISSUE #16: Add Exoscale setup documentation

### PR DESCRIPTION
**Pull request by timovdk**
Closes #16

### CODE REVIEW

File: asreview2-opt/README.md [hunk 1]
```diff
@@ -28,4 +28,29 @@
...
+10. Set your simulation parameters in `main.py` using a cli editor such as `nano main.py`
+11. Create a tmux environment so optuna keeps running when we close the connection `tmux new -s optuna`
+    In the `optuna` tmux env run the following commands to start the study:
+    1. `source .venv/bin/activate`
+    2. `python3 main.py`
+    3. Deattach from the tmux environment using `ctrl` `+` `b` followed by `d` (you can always reattach using `tmux attach -t optuna`)
+12. Create a tmux environment for the dashboard `tmux new -s dashboard`
+    In the `dashboard` tmux env run the following commands to start the study:
+    2. `optuna-dashboard sqlite:///db.sqlite3 --host 0.0.0.0`
+    3. Deattach from the tmux environment using `ctrl` `+` `b` followed by `d` (you can always reattach using `tmux attach -t dashboard`)
```
Comment (timovdk) [inline asreview2-opt/README.md:54]:
```suggestion
    1. `optuna-dashboard sqlite:///db.sqlite3 --host 0.0.0.0`
    2. Deattach from the tmux environment using `ctrl` `+` `b` followed by `d` (you can always reattach using `tmux attach -t dashboard`)
```

File: asreview2-opt/README.md [hunk 2]
```diff
@@ -28,4 +28,29 @@
...
+6. Create Python venv `python3 -m venv .venv`
+7. Activate venv `source .venv/bin/activate`
+8. Install Python packages `pip3 install -r requirements.txt`
+9. Create dataset pickles `python3 feature_matrices.py` (± 1.5 minutes)
+10. Set your simulation parameters in `main.py` using a cli editor such as `nano main.py`
+11. Create a tmux environment so optuna keeps running when we close the connection `tmux new -s optuna`
+    In the `optuna` tmux env run the following commands to start the study:
+    1. `source .venv/bin/activate`
+    2. `python3 main.py`
+    3. Deattach from the tmux environment using `ctrl` `+` `b` followed by `d` (you can always reattach using `tmux attach -t optuna`)
```
Comment (timovdk) [inline asreview2-opt/README.md:50]:
```suggestion
    3. Detach from the tmux environment using `ctrl` `+` `b` followed by `d` (you can always reattach using `tmux attach -t optuna`)
```

File: asreview2-opt/README.md [hunk 3]
```diff
@@ -28,4 +28,29 @@
...
+12. Create a tmux environment for the dashboard `tmux new -s dashboard`
+    In the `dashboard` tmux env run the following commands to start the study:
+    1. `optuna-dashboard sqlite:///db.sqlite3 --host 0.0.0.0`
+    2. Deattach from the tmux environment using `ctrl` `+` `b` followed by `d` (you can always reattach using `tmux attach -t dashboard`)
```
Comment (timovdk) [inline asreview2-opt/README.md:54]:
```suggestion
    2. Detach from the tmux environment using `ctrl` `+` `b` followed by `d` (you can always reattach using `tmux attach -t dashboard`)
```

**Comment by J535D165** [comment]
Very welcome!

**Pull request merged**